tools/osbuilder/rootfs-builder/cbl-mariner/config.sh

```diff
@@ -5,6 +5,6 @@
 OS_NAME=cbl-mariner
 OS_VERSION=${OS_VERSION:-2.0}
 LIBC="gnu"
-PACKAGES="core-packages-base-image ca-certificates device-mapper"
+PACKAGES="core-packages-base-image ca-certificates device-mapper opa"
```

Comment:
Is this a relatively urgent change?

If not, I think we should wait for https://github.com/kata-containers/kata-containers/pull/7574 to be merged first.

As part of that upstream pull request, I also fixed a bug on the libseccomp line below.